app/lib/dttp/entity/contact.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Dttp
+  module Entity
```

Comment:
I'm thinking of renaming `Entity` to 'Params`. Let me know if this is a good idea.